docs/developer-guide/maps-configuration.md

```diff
@@ -113,6 +113,19 @@ i.e.
 !!! warning
     Actually the custom resolution values are valid for one single CRS. It's therefore suggested to avoid to add this parameter when multiple CRSs in the same map configuration are needed.
 
+## Additional map configuration options
+
+Map configuration also contains the following additional options:
+
+- `catalogServices` object describing services configuration for Catalog
+- `widgetsConfig` configuration of map widgets
+- `mapInfoConfiguration` map info configuration options
+- `dimensionData` contains map time information
+    - `currentTime` currently selected time; the beginning of a time range if offsetTime is set
+    - `offsetTime` the end of a time range
+- `timelineData` timeline options
+    - `selectedLayer` selected layer id; if not present, the first available layer with multidim-extension time information will be selected on map load
```

Comment:
If the layer was not selected (snap disabled) you should not select the first layer, or the time range saved snaps to the nearest value.

Reply:
@offtherailz The reason for this is because setupTimelineExistingSettings epic checks for selectedLayer and if none present it selects the first layer with time info and snaps time. I didn't want to change this functionality in case it breaks anything.

Reply:
`setupTimelineExistingSettings` wants to auto select a layer on first "addLayer" when the layer is a time layer (forthis ). 
Also on REMOVE_NODE it changes the selected layer to clean up, if the layer removed was selected and auto select the next one.
In both cases it was snapping to now.
It was ok for the old behaviour. 
I think it have to be renew to follow these new setup rules. I think you could replace UPDATE_LAYER_DIMENSION_DATA action catch with a new action, TIMELINE;AUTOSELECT and trigger it in the proper places: 
 - after add_layer + load time data 
 - after map load if timelineData is not present (so it continues with the default behaviour.

Do you think this does the job? If so please test add layer, remove layer and other action sequence you consider important